contribution/lang/de.json

```diff
@@ -1153,15 +1153,15 @@
  	 	 	},
  	 	 	"possibleReasons": {
  	 	 	 	"1": {
- 	 	 	 	 	"trans": "Dies könnte verursacht werden durch:",
+ 	 	 	 	 	"trans": "Das könnte verursacht worden sein durch:",
```

Comment:
In this case I tried to translate 'This could have been caused by:' which then would be fine with 'worden' in there, but without it, it seems incorrect to me. Then I should maybe change it completely to 'Verursacht werden könnte das durch:'
What do you think about this?